docs/source/config.rst

```diff
@@ -452,6 +452,10 @@ Postprocessing
       syncing. Similarly to ``remove_local_files``, protected paths will not be 
       deleted.
 
+   ``runlog`` (*Default:* ``False``)
```

Comment:
I think it would be better to have this `True` by default

Reply:
Ok, I've just updated it to default to `True`